templates/default/ndbmtd.service.erb

```diff
@@ -14,6 +14,7 @@ ExecReload = <%= "#{node['ndb']['base_dir']}/scripts/ndbd-restart.sh" %>
 TimeoutSec= <%= node['ndb']['ndbd']['systemctl_timeout_sec'] %>
 Restart=on-failure
 RestartSec=1
+LimitMEMLOCK=infinity
```

Comment:
could this cause problems in the future

Reply:
I don't think so. We are setting a property for Datanodes to lock their memory but because we didn't give the process permission it was failing silently.